internal/kokoro/continuous.sh

```diff
@@ -101,6 +101,10 @@ runDirectoryTests() {
     # aliasshim: build constraints exclude all Go files
     return
   fi
+  if [[ $PWD == *"/benchmark" ]]; then
```

Comment:
The skip should apply only to dirs in which all .go files are excluded by build directives. `storage/internal/benchmarks` appears OK due to the presence of .go files that are not excluded by build directives.

Reply:
changed to exclude just for the bigquery case specifically